Source/WebCore/platform/graphics/win/ComplexTextControllerUniscribe.cpp

```diff
@@ -104,7 +104,6 @@ class BidiRange {
         void operator++() { ++m_logicalIndex; }
         T& operator*() { return m_range.m_data[index()]; }
         bool operator==(const Iterator& other) { return m_logicalIndex == other.m_logicalIndex; }
-        bool operator!=(const Iterator& other) { return m_logicalIndex != other.m_logicalIndex; }
```

Comment:
This one seems to have broken the Windows build?

Reply:
Ah, the operator is not marked as `const`. I'll fix.

Reply:
So glad you figured that one out! I didn’t spot the real issue and somehow thought it was a compiler difference.